src/modules/displaymanager/main.py

```diff
@@ -147,19 +176,40 @@ def set_autologin(username, displaymanagers, default_desktop_environment, root_m
                 else:
                     gdm_conf.write('AutomaticLoginEnable=False\n')
 
-        if do_autologin and os.path.exists("{!s}/var/lib/AccountsService/users".format(root_mount_point)):
-            os.system("echo \"[User]\" > {!s}/var/lib/AccountsService/users/{!s}".format(root_mount_point, username))
+        if (do_autologin
+                and os.path.exists("{!s}/var/lib/AccountsService/users".format(
+                    root_mount_point
+                    )
+                )):
+            os.system(
+                "echo \"[User]\" > "
+                + "{!s}/var/lib/AccountsService/users/{!s}".format(
```

Comment:
This is one of those cases where the + is superfluous and we could go for multiple string literals again, but since we've already agreed that this bit needs a re-write on account of using echo(1), let's leave it.